src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionAddActionTest.java

```diff
@@ -674,7 +669,21 @@ public void testExecuteAndPostProcessRubricQuestion() {
                 Const.ParamsNames.FEEDBACK_QUESTION_EDITTYPE, "edit",
         };
 
-        InstructorFeedbackQuestionAddAction action = getAction(params);
+        ______TS("Typical case with attached weights");
+
+        // tests when descriptions are missing as well.
```

Comment:
What is the comment for?
If it doesn't fit here, we can delete it.

Reply:
It was there before, so I didn't delete it. I guess, as the rubric question in the test has no description for the choices, this line has been added for that.

Okay, will delete it.